lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -51,7 +51,9 @@ describe('Fraggle Rock API', () => {
     server = new StaticServer();
     await server.listen(0, '127.0.0.1');
     serverBaseUrl = `http://localhost:${server.getPort()}`;
+    if (!process.env.CHROME_PATH) throw Error('CHROME_PATH is not set');
     browser = await puppeteer.launch({
+      executablePath: process.env.CHROME_PATH,
```

Comment:
`Audits` CDP domain is not available in the Chromium which ships with our current puppeteer.

Reply:
I think we can land #12284 soon.

Reply:
+1 to waiting for #12284 and/or just marking as an OK failure. #12510 has the same problem

Reply:
Should be good now.